bundles/org.openhab.binding.linky/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -35,18 +49,124 @@
 				<description>Authentication ID delivered after the captcha (see documentation).</description>
 			</parameter>
 		</config-description>
+
+	</bridge-type>
+
+
+	<bridge-type id="my-electrical-data">
+		<label>MyElectricalDataBridge</label>
+		<description>
+			Provides your energy consumption data.
+			In order to receive the data, you must activate your account at
+			https://espace-client-particuliers.enedis.fr/web/espace-particuliers/compteur-linky.
+		</description>
+
+		<config-description>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="linky">
+
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="enedis"/>
+			<bridge-type-ref id="enedis-web"/>
+			<bridge-type-ref id="my-electrical-data"/>
+		</supported-bridge-type-refs>
+
+		<label>Linky</label>
+		<description>
+			Provides your energy consumption data.
+			In order to receive the data, you must activate your account at
+			https://espace-client-particuliers.enedis.fr/web/espace-particuliers/compteur-linky.
+		</description>
+
```

Comment:
This  file need a bit of cleanup to remove excedent empty string. Did you run 'mvn spotless:apply' ?

Reply:
Yes, already done spotless:apply multiple time, but not sure it remove empty lines in this file !
I do a manual pass on it to clean it up.